### PR TITLE
リノートメニューに「リノートの詳細」を追加

### PR DESCRIPTION
**Pull request by FineArchs**
## What

renote-detailed

## Why

#14958
## Checklist
- [x] Read the [contribution guide](https://github.com/misskey-dev/misskey/blob/develop/CONTRIBUTING.md)
- [x] Test working in a local environment
- [ ] (If needed) Add story of storybook
- [x] (If needed) Update CHANGELOG.md
- [ ] (If possible) Add tests

### CODE REVIEW

File: packages/frontend/src/components/MkNote.vue [hunk 1]
```diff
@@ -565,16 +566,24 @@ function showRenoteMenu(): void {
 			},
 		};
 	}
+	const renoteDetailsMenu: MenuItem = {
```
Comment (kakkokari-gtyih) [inline packages/frontend/src/components/MkNote.vue:570]:
```suggestion

	const renoteDetailsMenu: MenuItem = {
```

File: packages/frontend/src/components/MkNote.vue [hunk 2]
```diff
@@ -565,16 +566,24 @@ function showRenoteMenu(): void {
...
+		type: 'link',
+		text: i18n.ts.renoteDetails,
+		icon: 'ti ti-info-circle',
+		to: notePage(note.value),
+	}
```
Comment (kakkokari-gtyih) [inline packages/frontend/src/components/MkNote.vue]:
```suggestion
	};
```

**Review by kakkokari-gtyih — COMMENTED** [review]
lint

**Comment by syuilo** [comment]
Changelogの位置がおかしそう

**Comment by syuilo** [comment]
🙏

**Pull request merged**